lib/util/util.c

```diff
@@ -2702,6 +2702,7 @@ bool util_ftruncate(FILE * stream , long size) {
 */
 
 bool util_same_file(const char * file1 , const char * file2) {
+#ifdef HAVE_UNISTD_H
```

Comment:
This: https://github.com/Statoil/libecl/blob/5748593069787de71e2d2ebbb0832efb0320676d/lib/include/ert/util/util.h#L377 is the symbol you should use.